<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stereograma!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Set Preset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="presetSelect"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="PresetEdit" name="presetEdit" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="renderButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Render</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="parallelProgress">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="ClickableLabel" name="rolling">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="rsc.qrc">:/images/rolling.jpg</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Get my free android game</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="PresetSketch" name="presetSketcher" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="ImageContainerWidget" name="helpers_image">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Eye Helpers</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QCheckBox" name="show_helpers">
            <property name="text">
             <string>Show Helpers?</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dual_helpers">
            <property name="text">
             <string>Dual Helpers?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="margin_helpers">
            <property name="text">
             <string>Helpers on margin</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="ImageContainerWidget" name="textureImage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>120</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Texture</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageContainerWidget" name="depthImage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>120</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Depth Map</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageContainerWidget" name="composeImage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>120</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Compose</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>compose 3d height</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="composeLevel">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>compose scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="composeScale">
        <property name="minimum">
         <number>64</number>
        </property>
        <property name="maximum">
         <number>768</number>
        </property>
        <property name="singleStep">
         <number>64</number>
        </property>
        <property name="pageStep">
         <number>64</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport_model">
     <property name="title">
      <string>Import model</string>
     </property>
     <addaction name="action_obj"/>
     <addaction name="action_ply"/>
    </widget>
    <addaction name="actionLoad_Texture"/>
    <addaction name="actionLoad_Depth_Map"/>
    <addaction name="menuImport_model"/>
    <addaction name="actionGenerate_Depth_Map"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPresets">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Presets</string>
    </property>
    <addaction name="actionAdd_preset"/>
    <addaction name="actionRename_Preset"/>
    <addaction name="actionRemove_preset"/>
    <addaction name="actionSave_Presets"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuAnaglyph">
    <property name="title">
     <string>Anaglyph</string>
    </property>
    <addaction name="actionCreate_Anaglyph"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPresets"/>
   <addaction name="menuAnaglyph"/>
  </widget>
  <action name="actionLoad_Texture">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/images/opent.png</normaloff>:/images/opent.png</iconset>
   </property>
   <property name="text">
    <string>Load Texture</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLoad_Depth_Map">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/images/opend.png</normaloff>:/images/opend.png</iconset>
   </property>
   <property name="text">
    <string>Load Depth Map</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAdd_preset">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add Preset</string>
   </property>
  </action>
  <action name="actionRemove_preset">
   <property name="text">
    <string>Remove Preset</string>
   </property>
  </action>
  <action name="action_obj">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/images/openobj.png</normaloff>:/images/openobj.png</iconset>
   </property>
   <property name="text">
    <string>Wavefront (.obj)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_ply">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/images/openobj.png</normaloff>:/images/openobj.png</iconset>
   </property>
   <property name="text">
    <string>Stanford PLY (.ply)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCreate_Anaglyph">
   <property name="text">
    <string>Create Anaglyph</string>
   </property>
  </action>
  <action name="actionSave_Presets">
   <property name="text">
    <string>Save Presets</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionRename_Preset">
   <property name="text">
    <string>Rename Preset</string>
   </property>
  </action>
  <action name="actionGenerate_Depth_Map">
   <property name="text">
    <string>Generate Depth Map</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageContainerWidget</class>
   <extends>QLabel</extends>
   <header>imagecontainerwidget.h</header>
  </customwidget>
  <customwidget>
   <class>PresetSketch</class>
   <extends>QWidget</extends>
   <header>presetsketch.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PresetEdit</class>
   <extends>QWidget</extends>
   <header>presetedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ClickableLabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
